data-products/src/main/resources/masterdata-indexer.conf

```diff
@@ -0,0 +1,20 @@
+# do not delete this file
+env=local
+
+redis.host="localhost"
+redis.port="6379"
+redis_scan_count=1000
+redis_maxPipelineSize=1000
+cloudStorage.container="/home/sankethika/obsrv-data"
```

Comment:
I think you can define the empty path and add the comment to include the local dir path. 
Also can we generalise the variables if we are using it for both local and cloud?

Reply:
Ok, I will change it.